### ISSUE TITLE
Error on click with leaflet v1.0.0-rc.1 release

### ISSUE THREAD

**Issue by YuanxiangFranck** [iosphere/Leaflet.hotline#6]
Leaflet hotline works fine but there is this error en each click:

`leaflet-src.js:8755: Uncaught TypeError: Cannot read property 'contains' of undefined`
```javascript
L.Polyline.prototype._containsPoint = function (p, closed) {
	var i, j, k, len, len2, part,
	    w = this._clickTolerance();

	if (!this._pxBounds.contains(p)) { return false; } // Error here this._pxBounds is undefined

	[...]
```
This is because between the version v1.0.0-rc.1 release and v1.0.0-beta.2 the _project function changed.

```javascript
_project: function () {
	var pxBounds = new L.Bounds();
	this._rings = [];
	this._projectLatlngs(this._latlngs, this._rings, pxBounds);
	var w = this._clickTolerance(),
	    p = new L.Point(w, w);
	if (this._bounds.isValid() && pxBounds.isValid()) { // pxBounds.isValid() is false
		pxBounds.min._subtract(p);
		pxBounds.max._add(p);
		this._pxBounds = pxBounds;
	}
},
```
So this.pxBounds is not defined!

**Comment by klaftertief** [comment]
Thanks for the detailed report.
I'll go over the plugin to update it to the latest Leaflet RC soonish.

**Comment by KendallPark** [comment]
I'm encountering this as well. A fix would be much appreciated! :D 

@YuanxiangFranck any hack to get around this for now?

**Comment by klaftertief** [comment]
It's fixed in version [0.3.0](https://github.com/iosphere/Leaflet.hotline/releases/tag/v0.3.0).

**Comment by KendallPark** [comment]
👍 If you are ever in Columbia, MO I will buy you a 🍺 .
